templates/cv_entry.md

```diff
@@ -1,44 +1,53 @@
-<!-- TODO: Create the heading Pt(111) from metadata. See #31. -->
 <!-- TODO: Create the small heading subtitle from metadata. See #31. -->
-# Pt(111) <small>- 0.1 M CsF + 0.1 M HClO4</small>
+# {{ entry.electrochemical_system.electrodes.working_electrode.material }}({{ entry.electrochemical_system.electrodes.working_electrode.crystallographic_orientation }}) <small>- 0.1 M CsF + 0.1 M HClO4</small>
```

Comment:
I don't understand the science here but I wonder if this is so important that it should be made available directly on the `entry`? I.e., add a property `material` (or something like that) that returns "Pt(111)".

In particular, this might be something that people want to filter the database on if I understood correctly.

Reply:
One could think about filtering directly `Pt(111)`, then a common entry for both keys would be beneficial.
`material` is too generic, since the other electrodes in the set-up are also made up of some kind of material.
We could use a property `working_electrode`, if this does not become too confusing with the other `working_electrode` key. Maybe @linuxrider or @nicohoermann have another idea for this issue.

Reply:
Another fancy option would be to make `entry.electrochemical_system.electrodes.working_electrode` render as `Pt(111)` when asked to convert to string.

Reply:
We can of course postpone this to an issue.

Reply:
see #42